Modelica/package.mo

```diff
@@ -8698,25 +8698,18 @@ revisionId="$Format:%h %ci$",
-    smooth=Smooth.Bezier),
-  Ellipse(
-    origin={40.8208,-37.7602},
-    fillColor={161,0,4},
-    pattern=LinePattern.None,
-    fillPattern=FillPattern.Solid,
-    extent={{-17.8562,-17.8563},{17.8563,17.8562}})}),
-Documentation(info="<html>
+Icon(graphics={Polygon(origin = {-7, 20}, fillColor = {112, 122, 132}, pattern = LinePattern.None, fillPattern = FillPattern.Solid, points = {{-93.0112, 10.3188}, {-93.0112, 10.3188}, {-73.011, 24.6}, {-63.011, 31.221}, {-51.219, 36.777}, {-39.842, 38.629}, {-31.376, 36.248}, {-25.819, 29.369}, {-24.232, 22.49}, {-23.703, 17.463}, {-15.501, 25.135}, {-6.24, 32.015}, {3.02, 36.777}, {15.191, 39.423}, {27.097, 37.306}, {32.653, 29.633}, {35.035, 20.108}, {43.501, 28.046}, {54.085, 35.19}, {65.991, 39.952}, {77.897, 39.688}, {87.422, 33.338}, {91.126, 21.696}, {90.068, 9.525}, {86.099, -1.058}, {79.749, -10.054}, {71.283, -21.431}, {62.816, -33.337}, {60.964, -32.808}, {70.489, -16.14}, {77.368, -2.381}, {81.072, 10.054}, {79.749, 19.05}, {72.605, 24.342}, {61.758, 23.019}, {49.587, 14.817}, {39.003, 4.763}, {29.214, -6.085}, {21.012, -16.669}, {13.339, -26.458}, {5.401, -36.777}, {-1.213, -46.037}, {-6.24, -53.446}, {-8.092, -52.387}, {-0.684, -40.746}, {5.401, -30.692}, {12.81, -17.198}, {19.424, -3.969}, {23.658, 7.938}, {22.335, 18.785}, {16.514, 23.283}, {8.047, 23.019}, {-1.478, 19.05}, {-11.267, 11.113}, {-19.734, 2.381}, {-29.259, -8.202}, {-38.519, -19.579}, {-48.044, -31.221}, {-56.511, -43.392}, {-64.449, -55.298}, {-72.386, -66.939}, {-77.678, -74.612}, {-79.53, -74.083}, {-71.857, -61.383}, {-62.861, -46.037}, {-52.278, -28.046}, {-44.869, -15.346}, {-38.784, -2.117}, {-35.344, 8.731}, {-36.403, 19.844}, {-42.488, 23.813}, {-52.013, 22.49}, {-60.744, 16.933}, {-68.947, 10.054}, {-76.884, 2.646}, {-93.0112, -12.1707}, {-93.0112, -12.1707}, {-93.0112, 10.3188}}, smooth = Smooth.Bezier), Ellipse(origin = {40, -40}, fillColor = {222, 29, 49}, pattern = LinePattern.None, fillPattern = FillPattern.Solid, extent = {{-18, -18}, {18, 18}}, endAngle = 360)}),
+Documentation(info= "<html>
 <p>
-Package <strong>Modelica&reg;</strong> is a <strong>standardized</strong> and <strong>free</strong> package
-that is developed together with the Modelica&reg; language from the
-Modelica Association, see
-<a href=\"https://www.Modelica.org\">https://www.Modelica.org</a>.
+<img src=\"modelica://Modelica/Resources/Images/Logos/Modelica_Libraries.svg\" style=\"width:250\">
+</p>
+
+<p>
+The package <strong>Modelica&reg;</strong> is a <strong>standardized</strong> and <strong>free</strong> package
+that is developed by the \"<strong>Modelica Association Project - Libraries</strong>\".</p>
+<p>
+Its development is coordinatted with with the Modelica&reg; language from the
```

Comment:
I think this should be `coordinated with the`, with a single `t` and a single `with`

Reply:
Fixed in 1b4507b6e051fabf62d0f738c30547e3256ffecb